cli/__tests__/cli-integration.test.ts

```diff
@@ -132,6 +140,18 @@ for (const packageManager of packageManagers) {
         await Bun.$`find ./${projectName} -not -path "./${projectName}/node_modules*" -not -path "./${projectName}/.git*" | sort`.text();
 
       expect(fileList).toMatchSnapshot(`${finalFlags.join(', ')}-file-list`);
+
+      // once nwui cli text component is fixed we can remove this check
+      if (!finalFlags.includes('--nativewindui')) {
```

Comment:
we need the uiTextView to be default for 'text' so that the types aren't broken, then we can remove this check